.travis.yml

```diff
@@ -0,0 +1,25 @@
+sudo: required
```

Comment:
Do you use travis here or is that an old artefact?

Reply:
Yes, I think this is an old artefact. I'll remove it the next commit.